doc/whats_new.rst

```diff
@@ -67,6 +67,8 @@ Bug
 - Fix :func:`mne_bids.write_raw_bids` failed BIDS validator for ``raw.info['dig'] = []`` by `Alex Rockhill`_ (`#505 <https://github.com/mne-tools/mne-bids/pull/505>`_)
 - Fix bug in :func:`mne_bids.BIDSPath.fpath` where no data file was returned when the suffix was not assigned, by `Alex Rockhill`_ (`#542 <https://github.com/mne-tools/mne-bids/pull/542>`_)
 - Ensure :func:`mne_bids.print_dir_tree` prints files and directories in alphabetical order, by `Richard Höchenberger`_ (`#563 <https://github.com/mne-tools/mne-bids/pull/563>`_)
+- Ensure :func:`mne_bids.write_raw_bids` writes events encoded in annotations of the raw file, by `Adam Li`_ (`#582 <https://github.com/mne-tools/mne-bids/pull/582>`_)
```

Comment:
```suggestion
- :func:`mne_bids.write_raw_bids` now also writes  :class:`mne.Annotations` if present in the raw file, by `Adam Li`_ (`#582 <https://github.com/mne-tools/mne-bids/pull/582>`_)
```

Reply:
This is an enhancement / new feature, not a bug fix, as we never claimed to preserve Annotations. Can you move it to the respective section?